README.md

```diff
@@ -103,6 +104,60 @@ The following snippet shows an example `build.gradle` with APK splits enabled.
 
 The adoption of APK splits requires developers to submit multiple APKs to the Play Store. Refer to [Google’s documentation](https://developer.android.com/google/play/publishing/multiple-apks.html) for how to support this in your application.
 
+## Troubleshooting Audio
+The following sections provide guidance on how to ensure optimal audio quality in your applications.
+
+### Configuring [AudioManager](https://developer.android.com/reference/android/media/AudioManager.html)
```

Comment:
Would prefer moving the link to `AudioManager` to 111 instead of having the link on the header